### PR TITLE
Oropouche blog post

### PR DESCRIPTION
**Pull request by miparedes**
## Description of proposed changes

This PR adds in the blogpost for the new oropouche build as well as the associated images.

## Checklist

- [ ] Checks pass
- [ ] Check if changes affect the [resource index JSON revision](https://docs.nextstrain.org/projects/nextstrain-dot-org/en/latest/resource-collection.html#resource-index-revisions)

### CODE REVIEW

**Review by jameshadfield — APPROVED** [review]

File: static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+author: "Miguel Paredes, James Hadfield, John Anderson, Jover Lee, Trevor Bedford"
```
Comment (genehack) [inline static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md:2]:
```suggestion
author: "Miguel Paredes, James Hadfield, John SJ Anderson, Jover Lee, Trevor Bedford"
```

File: static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+date: "2024-10-18"
+title: "Phylogenetic analysis of Oropuche Virus"
```
Comment (genehack) [inline static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md:4]: not sure what the prevailing style is across current blog posts, but I would suggest either title-casing the whole thing, or down-casing `"Virus"`...

File: static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+sidebarTitle: "Oropouche Phylogenetic Analysis and Resources"
+---
+
+# Phylogenetic analysis
+
+[Oropouche virus](https://www.sciencedirect.com/science/article/pii/S016817022400011X#sec0003) is a segmented, negative-sense, single-stranded RNA virus from the Orthobunyavirus genus. It is the causative agent of Oropouche fever: an often self-limited, non-specific disease characterized by headache, arthralgia, myalgia, nausea, vomiting, chills, and/or photophobia. Oropouche virus is maintained primarily through an enzootic cycle between pale-throated sloths (Bradypus tridactylus), non-human primates and other wild mammals via transmission through Culicoides paraensis midges, with urbanization contributing to the rise of human cases acquired through the same arthropod vector (**Fig. 1**).
```
Comment (genehack) [inline static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md:10]:
```suggestion
[Oropouche virus](https://www.sciencedirect.com/science/article/pii/S016817022400011X#sec0003) is a segmented, negative-sense, single-stranded RNA virus from the Orthobunyavirus genus. It is the causative agent of Oropouche fever: an often self-limited, non-specific disease characterized by headache, arthralgia, myalgia, nausea, vomiting, chills, and/or photophobia. Oropouche virus is maintained primarily through an enzootic cycle between pale-throated sloths (_Bradypus tridactylus_), non-human primates and other wild mammals via transmission through _Culicoides paraensis_ midges, with urbanization contributing to the rise of human cases acquired through the same arthropod vector (**Fig. 1**).
```

**Review by genehack — APPROVED** [review]
+1 in general, along with a suggestion to perhaps add a short intro paragraph at the very beginning that explicitly says something like, "we are pleased to announce the availability of a Oropouche phylogeny dataset on Nextstrain.org" and maybe also work in a "we are interested in feedback from Oropouche researchers about any aspects of this data set"

File: static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md (hunk 2 above)
Comment (miparedes) [reply static-site/content/blog/2024-06-18-oropouche-analysis-and-resources.md]: agreed! I updated this in the most recent commit, and suggestion actually highlighted out to me that "Oropouche" is mispelled above so I went ahead and fixed that. I also added in an intro paragraph as per your suggestion

**Review by trvrb — APPROVED** [review]
I just made some small fixes to screenshots and URL linkouts. This is good to be published now from my perspective.

File: static-site/content/blog/2024-10-22-oropouche-analysis-and-resources.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+M segment
+
+- [data.nextstrain.org/files/workflows/oropouche/M/sequences.fasta.zst](data.nextstrain.org/files/workflows/oropouche/M/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/oropouche/M/metadata.tsv.zst](data.nextstrain.org/files/workflows/oropouche/M/metadata.tsv.zst)
+
+S segment
+
+- [data.nextstrain.org/files/workflows/oropouche/S/sequences.fasta.zst](data.nextstrain.org/files/workflows/oropouche/S/sequences.fasta.zst)
+- [data.nextstrain.org/files/workflows/oropouche/S/metadata.tsv.zst](data.nextstrain.org/files/workflows/oropouche/S/metadata.tsv.zst
+)
```
Comment (joverlee521) [inline static-site/content/blog/2024-10-22-oropouche-analysis-and-resources.md:62]: _Totally missed this PR_ 🙈 

The structure of the underlying data changed to 1 metadata TSV + 3 FASTAs in https://github.com/nextstrain/oropouche/pull/18. 

See the latest description at https://github.com/nextstrain/oropouche/blob/main/phylogenetic/defaults/description.md#underlying-data

Comment (joverlee521) [reply static-site/content/blog/2024-10-22-oropouche-analysis-and-resources.md:62]: (I was wondering why the [live build](https://next.nextstrain.org/oropouche/L) footer hasn't been updated and it's due to the [cache not updating](https://github.com/nextstrain/oropouche/issues/20))

Comment (joverlee521) [reply static-site/content/blog/2024-10-22-oropouche-analysis-and-resources.md:62]: Links were updated in https://github.com/nextstrain/nextstrain.org/pull/1046

**Pull request merged**